Source/Diagnostics/BTDiagnostics.H

```diff
@@ -9,7 +9,8 @@
 
 #include "Diagnostics.H"
 #include "Diagnostics/ComputeDiagFunctors/ComputeDiagFunctor.H"
-#include "Utils/WarpXConst.H"
+#include "Utils/IntervalsParser.H"
+#include <string>
```

Comment:
Is `#include "Utils/WarpXConst.H"` not used anymore?

Please include `<string>` last: https://warpx.readthedocs.io/en/latest/developers/repo_organization.html#c-includes

Reply:
the deletion is an accident... still learning how to effectively do in-line commits from github